easyrest/models/user.py

```diff
@@ -70,3 +70,12 @@ def add(database, form_data):
         password_hash = pbkdf2_sha256.hash(password)
 
         database.add(User(name=name, email=email, password=password_hash))
+
+    @staticmethod
+    def delete(database, user):
+        """This function removes the user and his data from the database.
+
+        :param database: Database session.
+        :param user: Instance of deletable user.
+        """
+        database.delete(user)
```

Comment:
In this particular case you can do delete method like this:

def delete(self, session):
      session.delete(self)

So no need to write @staticmethod

Reply:
It makes sense. Thank's.

Reply:
Its confusing)

Reply:
Guess which function deletes the user?
Function 1:
```
def delete(self, session):
    session.delete(self)
```
Function 2:
```
def delete(database, user):
    database.delete(user)
```

Reply:
I made this function with some expectation for the future. For example, if you need to delete files associated with a user (well, avatar or something else), you can simply do something like this to 
the delete method:
```
def delete(database, user):
    database.delete(user)
    storage.delete(user.avatar)
```